workflows/iuf/operations/management-nodes-rollout/management-nodes-rollout.yaml

```diff
@@ -98,23 +98,18 @@ spec:
                     if [[ -n $(echo $limit_management_nodes | grep 'Management_Master') ]]; then rebuild_masters=true; fi
 
                     prepare_images_ouput=$(echo '{{inputs.parameters.global_params}}' | jq -r '.stage_params["prepare-images"].global["prepare-management-images"]["sat-bootprep-run"].script_stdout')
-                    update_cfs_conf_ouput=$(echo '{{inputs.parameters.global_params}}' | jq -r '.stage_params["update-cfs-config"].global["update-management-cfs-config"]["sat-bootprep-run"].script_stdout')
-
-                    # check config
-                    configuration=$(echo $update_cfs_conf_ouput | jq '.configurations[].name' | tr -d '"')
-                    if [[ -z $configuration ]]; then echo "ERROR  no CFS configuration was recieved from update-cfs-config stage"; exit 1; fi
-                    if [[ $(echo $configuration | wc -l) -gt 1 ]]; then 
-                      echo "ERROR  more than 1 configruation was recieved. There must be exactly 1 CFS configuration to for a rebuild"
-                      echo "ERROR  CFS configurations recieved: $configuration"; exit 1
-                    fi
-                    cray cfs configurations describe "$configuration" > /dev/null
-                    if [[ $? -ne 0 ]]; then
-                      exit 1 # could not find the desired cfs configuration
-                    else
-                      echo "NOTICE found CFS configuration:$configuration in CFS"
-                    fi
 
                     if $rebuild_workers; then
+                      # Check config exists
+                      config=$(echo $prepare_images_ouput | jq '.images | map(select(.configuration_group_names[] == "Management_Worker")) | .[].configuration' | tr -d '"')
+                      if [[ -z $config ]]; then echo "ERROR  no CFS configuration was recieved for 'Management_Worker' from prepare images stage"; exit 1; fi
```

Comment:
I'm guessing the error here would more accurately be that there was no image for management_worker?